mgmtfn/pslibnet/main.go

```diff
@@ -67,7 +68,6 @@ var gcliOpts cliOpts
 func main() {
 	var flagSet *flag.FlagSet
 
-	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
I can look into how that’s implemented in logrus (if at all), but I am preparing a log levels patch right now.

Is it possible to merge this and revisit early next week?

> On May 21, 2015, at 5:16 PM, Madhav Puri <notifications@github.com> wrote:
> 
> In mgmtfn/pslibnet/main.go <https://github.com/contiv/netplugin/pull/76#discussion_r30861073>:
> 
> > @@ -67,7 +68,6 @@ var gcliOpts cliOpts
> >  func main() {
> >  	var flagSet *flag.FlagSet
> >  
> > -	log.SetFlags(log.LstdFlags | log.Lshortfile)
> I found setFlag() useful for easily modifying the log format (Example, having file name and line number comes in handy for debugging some failure without having to grep through code).
> 
> I think logrus doesn't add this info by default and we could potentially define a custom formatter, if we want that info in logs later?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/76/files#r30861073>.
>